prac_07/project.py

```diff
@@ -0,0 +1,14 @@
+class Project:
+    def __init__(self, name, start_date, priority, cost_estimate, percent_complete):
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.cost_estimate = cost_estimate
+        self.percent_complete = percent_complete
+
+    def __str__(self):
```

Comment:
__str__(self): This method returns a string representation of the Project object. It formats the output to include details about the project's name, start date, priority, cost estimate, and percent completion.